docs/src/quickstart/index.md

```diff
@@ -57,356 +70,184 @@ for these executables in your `$PATH`.
+Deploying index at tina.manifest.yaml to http://127.0.0.1:29987/api/index/my_project/hello_index
+▹▸▹▹▹ 🚀 Deploying...
 {
   "assets": [
     {
-      "digest": "d797e33a3d3bbc4d93c7ec09980c220b1243e0ffcd9107b6e13b61cb7704d5ec",
+      "digest": "79e74d6a7b68a35aeb9aa2dd7f6083dae5fdba5b6a2f199529b6c49624d1e27b",
       "id": 1,
       "index_id": 1,
       "version": 1
     },
     {
-      "digest": "c5af6d278e29efd47a0493de73509bf4329ca58e47d337d0cc368e0a1f110cb9",
+      "digest": "4415628d9ea79b3c3f1e6f02b1af3416c4d0b261b75abe3cc81b77b7902549c5",
       "id": 1,
       "index_id": 1,
       "version": 1
     },
     {
-      "digest": "b32879df38991e7b4f19ed02e394e2d31396cf1fa5ba14429e2af50dfca18cc7",
+      "digest": "e901eba95ce8b4d1c159c5d66f24276dc911e87dbff55fb2c10d8b371528eacc",
       "id": 1,
       "index_id": 1,
       "version": 1
     }
   ],
   "success": "true"
 }
-
-✅ Successfully deployed index at hello_index.manifest.yaml to http://127.0.0.1:29987/api/index/fuel/hello_index
+▪▪▪▪▪ ✅ Successfully deployed index.
 ```
 
-## Generating test data
+## 3. Querying for data
 
-Now that we've successfully deployed our index, let's make a few calls to our Sway contract in order to produce a few events, and index some data.
+With our index deployed, after a few seconds, we should be able to query for newly indexed data.
 
 ```bash
-
-# Go back to the repository root
-➜ cd fuel-indexer/
-
-# Run the test data generator for this example
-➜ cargo run --bin hello-bin
-```
-
-> One contract call will be made, and one event will be emitted to be indexed.
-You can continue running this command to generate more data.
-
-----
-
-## Querying for indexed events
-
-After you've successfully completed all six of the aforementioned steps, you can trigger some test events simply by calling the `new_greeting()` method of your Sway contract. This will produce blocks, transactions, and receipts, which will be emitted by your local Fuel node. These events will be picked up by the indexer and subsequently indexed according to the index that you've deployed. Once you have a few indexed events, you can query the indexer for the data that you wish to receive.
-
-### Query for all records of a type
-
-```sh
-➜ curl -X POST http://127.0.0.1:29987/api/graph/fuel_examples \
+➜ curl -X POST http://0.0.0.0:29987/api/graph/my_project \
    -H 'content-type: application/json' \
-   -d '{"query": "query { greeter { id name first_seen last_seen }}", "params": "b"}' \
+   -d '{"query": "query { tx { id hash status block }}", "params": "b"}' \
 | json_pp
   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                  Dload  Upload   Total   Spent    Left  Speed
 100   364  100   287  100    77   6153   1650 --:--:-- --:--:-- --:--:--  9100
 [
    {
-      "first_seen" : 4,
-      "id" : 2314885811371338051,
-      "last_seen" : 4,
-      "name" : "Ciara"
+      "block" : 7017844286925529648,
```

Comment:
Just an FYI: We'll need to revisit this after we implement foreign key reference object returns in our query responses.